docs/ext/bridge/index.rst

```diff
@@ -62,13 +62,8 @@ Bots
 .. autoclass:: discord.ext.bridge.AutoShardedBot
     :members:
 
-    .. automethod:: Bot.add_bridge_command()
-
-    .. automethod:: Bot.bridge_command()
-        :decorator:
-
 Commands
-~~~~~~~~
+---------
```

Comment:
Shouldn't the amount of symbols be the same as the length of the text above it?

Reply:
Generally, in most places that I've checked, the amount of symbols extends one past the length of text above it. I can change it if you like, but this was mainly done to keep consistency with other documentation files like `docs/api.rst`.

Reply:
Fair enough